test/Microsoft.DotNet.DarcLib.Tests/GitHubClientTests.cs

```diff
@@ -398,9 +398,9 @@ private async Task<IList<Review>> GetLatestReviewsForPullRequestWrapperAsync(Dic
 
         // Use Moq to put the return value 
         OctoKitPullRequestReviewsClient.Setup(x => x.GetAll(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<int>()))
-            .Callback<string, string, int>((string x, string y, int z) =>
+            .Callback((string x, string y, int z) =>
             {
-                Tuple<string, string, int> theKey = new Tuple<string, string, int>(x, y, z);
+                var theKey = new Tuple<string, string, int>(x, y, z);
```

Comment:
I believe having a type specified has more pros than cons.

Reply:
But the type is still there, just not twice in the same row?

Reply:
This only changes the "it's obvious" cases